From 1be9d3d04a25be268234eef05a8053abea09a03d Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Sat, 11 Feb 2017 00:26:37 -0800
Subject: [PATCH] ActivityManager: Silence "Failed setting process group"
 logspam

These logs are adding a lot of noise and they're trivial when cpusets
is enabled.

Change-Id: I529c4a97a4b4c2fbc71e75f8753dc439871e93e0
---
 .../core/java/com/android/server/am/ActivityManagerService.java   | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/services/core/java/com/android/server/am/ActivityManagerService.java b/services/core/java/com/android/server/am/ActivityManagerService.java
index 23f34e6..9418ea2 100755
--- a/services/core/java/com/android/server/am/ActivityManagerService.java
+++ b/services/core/java/com/android/server/am/ActivityManagerService.java
@@ -19199,9 +19199,11 @@ public final class ActivityManagerService extends ActivityManagerNative
                     try {
                         Process.setProcessGroup(app.pid, app.curSchedGroup);
                     } catch (Exception e) {
-                        Slog.w(TAG, "Failed setting process group of " + app.pid
-                                + " to " + app.curSchedGroup);
-                        e.printStackTrace();
+                        if (DEBUG_SWITCH || DEBUG_OOM_ADJ) {
+                            Slog.v(TAG, "Failed setting process group of " + app.pid
+                                    + " to " + app.curSchedGroup);
+                            e.printStackTrace();
+                        }
                     } finally {
                         Binder.restoreCallingIdentity(oldId);
                     }
-- 
2.9.3

